### ISSUE TITLE
`epi_slide` and `epix_slide` args

### ISSUE THREAD

**Issue by dajmcdon** [cmu-delphi/epiprocess#510]
I still find before/after confusing. It takes mental math and careful reading of the documentation. It's more general, but not really how our target users are likely to think.

I suggest we bring back `n` for the window size as well as `align = c("left", "center", "right")`. We can keep `before`/`after` as `NULL` and ensure there aren't any conflicts. Then immediately process `n`+`align` into the appropriate `before`+`after` pair.

**Comment by brookslogan** [comment]
Thanks for the ping.  I think this fell down the stack due to some engineering priorities + new testers getting tripped up by other things.  Pinging @dshemetov here in case he's interested in tackling; seems related to time type rework.

@dajmcdon have you ever used / remember us using something that's not left/center/right aligned?  I know we've used non-odd-`n` center-aligned things at least (e.g., `before = 6, after = 7`), but that's the closest I can think of.

Context/history for implementer:
- We didn't move to `before` & `after` for generality; `epi_slide` actually had an `n` + `align` + `before`, and you had to provide `n` plus exactly one of the latter two; this allowed for the same level of generality as `before` + `after`.  The actual reason for the change had to deal with `epix_slide` + trying to mirror interfaces between `epix_slide` and `epi_slide`; the latter's no longer a key goal. 
  - Plan since this summer has been to urgently change back to `n` + `align` + `before`.
  - Though `n` + `before` seems awkward; I like @dajmcdon's suggestion of `n` + `align` + `before` + `after`.... assuming we want to support this level of generality at all.
- We should make sure to mirror @dshemetov's time type type rework rules when implementing this change.
- (History: how did `epix_slide` make use change `epi_slide`?  `n` was always misleading in `epix_slide` to some people (and sometimes misleading in `epi_slide`, though mirroring the recent time type rework should take care of some cases; the remaining cases are the continually-frustrating/complicated cases with non-epikey-grouped epi_slides with differing epikey availability), so we changed to `epix_slide(before =)` and, for consistency, `epi_slide(before =, after =)`.)
